### PR TITLE
[FIXED JENKINS-31887] Added support for Selenium Report plugin

### PR DESCRIPTION
**Pull request by v1v**
https://issues.jenkins-ci.org/browse/JENKINS-31887

Plugin: https://wiki.jenkins-ci.org/display/JENKINS/Seleniumhq+Plugin

### CODE REVIEW

File: job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext.groovy [hunk 1]
```diff
@@ -1738,6 +1738,22 @@ class PublisherContext extends AbstractExtensibleContext {
         }
     }
 
+    /*
```
Comment (daspilker) [inline job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext.groovy]: The GroovyDoc comment block needs to start with `/**`. Fix this for all your pull requests.

Comment (v1v) [inline job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext.groovy]: Thanks for that spot! :S

**Comment by daspilker** [comment]
Add a releases notes entry, e.g.

```
 * Added support for the [Foo Plugin](https://wiki.jenkins-ci.org/display/JENKINS/Foo+Plugin)
   ([JENKINS-12345](https://issues.jenkins-ci.org/browse/JENKINS-12345))
```

And squash your commits to a single one.

Do this for all of your open pull requests.

**Comment by v1v** [comment]
Sure,

Thanks Daniel

**Pull request merged**